build.gradle

```diff
@@ -357,6 +372,7 @@ task testConnected(type: Test) {
         events = ["passed", "failed", "skipped", "started", "standard_out"]
     }
     outputs.upToDateWhen { false }
+    maxHeapSize = "2g"
```

Comment:
Maybe not in this PR (since we know things will run) but let's try to remember this in our collective consciousness in case we run into similar OOMs in unit and integration tests